code/asteroid/asteroid.cpp

```diff
@@ -2040,7 +2089,9 @@ static void asteroid_parse_tbl()
 
 			asteroid_parse_section(&new_asteroid);
 
-			maybe_change_asteroid_name(&new_asteroid);
+			// I really don't think this is needed and actually causes a bug in FRED
+			// making it impossible to select some debris types-Mjn
+			//maybe_change_asteroid_name(&new_asteroid);
```

Comment:
As mentioned, this is fine, but be sure to update the hud targetbox code https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/hud/hudtargetbox.cpp#L1246 to remove trailing nubmers, which is what I think what we agreed should be sufficient.

Reply:
I'll punt this to #4870 which re-introduces the whole feature with a distinct asteroid display name.